policy/pipeline/required_tasks_test.rego

```diff
@@ -21,27 +22,32 @@ test_required_tasks_not_met if {
 	pipeline := _pipeline_with_tasks_and_label(_expected_required_tasks - missing_tasks, [], [])
 
 	expected := _missing_tasks_violation(missing_tasks)
-	lib.assert_equal_results(expected, deny) with data["pipeline-required-tasks"] as _time_based_pipeline_required_tasks
+	lib.assert_equal_results(
+		expected,
+		required_tasks.deny,
+	) with data["pipeline-required-tasks"] as _time_based_pipeline_required_tasks
 		with input as pipeline
 }
 
 test_future_required_tasks_met if {
 	pipeline := _pipeline_with_tasks_and_label(_expected_future_required_tasks, [], [])
-	lib.assert_empty(warn) with data["pipeline-required-tasks"] as _time_based_pipeline_required_tasks
+	lib.assert_empty(required_tasks.warn) with data["pipeline-required-tasks"] as _time_based_pipeline_required_tasks
 		with input as pipeline
 
 	pipeline_finally := _pipeline_with_tasks_and_label([], _expected_future_required_tasks, [])
-	lib.assert_empty(warn) with data["pipeline-required-tasks"] as _time_based_pipeline_required_tasks
+	lib.assert_empty(required_tasks.warn) with data["pipeline-required-tasks"] as _time_based_pipeline_required_tasks
 		with input as pipeline_finally
 }
 
 test_not_warn_if_only_future_required_tasks if {
 	pipeline := _pipeline_with_tasks_and_label(_expected_future_required_tasks, [], [])
-	lib.assert_empty(warn) with data["pipeline-required-tasks"] as _time_based_pipeline_required_tasks_future_only
+	tasks1 = _time_based_pipeline_required_tasks_future_only
```

Comment:
I'm surprised it doesn't want: `tasks1 := ...`

Wait, why have `tasks` and `tasks2`? Can't we use the same in both cases?

Reply:
Been staring at this for too long I guess...